src/metabase/query_processor/pivot/impl/legacy.clj

```diff
@@ -0,0 +1,294 @@
+(ns metabase.query-processor.pivot.impl.legacy
```

Comment:
Everything in this namespace is moved from `metabase.query-processor.pivot` without any changes to the logic